wsgi/odyssy/odyssy/urls.py

```diff
@@ -30,5 +30,9 @@
     url(r'^news/', include('news.urls')),
     url(r'^tag/', include('tag.urls')),
     url(r'^announcement/', include('announcement.urls')),
+    url(r'.html/$', include('serve_static.urls')),
```

Comment:
To link the static files with the templates, I was having trouble linking them in the basic app itself. So @Prakash2403 sir had this idea of creating an app to serve the static files to templates without registering all the individual templates in url.